docs/source/contents/inference/index.md

```diff
@@ -1,107 +1,375 @@
+2. Determine the appropriate headers/metadata for the request.
+3. Make requests via REST or gRPC.
 
-### Find Seldon service endpoint
+### Find the Seldon Service Endpoint
 
- 1. If you are running Seldon locally via Docker compose by default the endpoint will be `0.0.0.0:9000`
- 2. If you are running in Kubernetes Seldon creates a single Service `seldon-mesh` in the namespace Seldon is installed to, usually `seldon-mesh`. If this has be exposed via a load balancer this can be found via:
+`````{tabs}
 
- ```bash
- kubectl get svc seldon-mesh -n seldon-mesh -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
- ```
+````{tab} Docker Compose
 
-### Make inference requests
+In the default Docker Compose setup, container ports are accessible from the host machine.
+This means you can use `localhost` or `0.0.0.0` as the hostname.
 
-Seldon routes requests to to the correct enpoint via headers in HTTP calls. You should set the header `seldon-model` as follow:
+The default port for sending inference requests to the Seldon system is `9000`.
+This is controlled by the `ENVOY_DATA_PORT` environment variable for Compose.
 
- * For Models: use the model name, e.g. for a Model names `mymodel`, `seldon-model: mymodel`
- * For Pipelines: use the Pipeline name with the suffix `.pipeline`, e.g. for a Pipeline named `mypipeline`, `seldon-model: mypipeline.pipeline`
+Putting this together, you can send inference requests to `0.0.0.0:9000`.
+````
 
-The content of your request should be a [V2 protocol payload](../apis/inference/v2.md).
+````{tab} Kubernetes
 
-The `seldon` CLI can be used to easily send requests to your deployed resources. See the [examples](../examples/index) and the [seldon CLI docs](../cli/index.md).
+In Kubernetes, Seldon creates a single `Service` called `seldon-mesh` in the namespace it is installed into.
+By default, this namespace is also called `seldon-mesh`.
 
-An example curl request might look like for a Model called `iris`:
+If this `Service` is exposed via a load balancer, the appropriate address and port can be found via:
 
+```bash
+kubectl get svc seldon-mesh -n seldon-mesh -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
 ```
-curl -v http://0.0.0.0:9000/v2/models/iris/infer -H "Content-Type: application/json" -H "seldon-model: iris"\
+
+If you are not using a `LoadBalancer` for the `seldon-mesh` `Service`, you can still send inference requests.
+
+For development and testing purposes, you can port-forward the `Service` locally using the below.
+Inference requests can then be sent to `localhost:8080`.
+
+```
+kubectl port-forward svc/seldon-mesh -n seldon-mesh 8080:80
+```
+
+If you are using a service mesh like Istio or Ambassador, you will need to use the IP address of the service mesh ingress and determine the appropriate port.
+````
+
+`````
+
+### Make Inference Requests
+
+Let us imagine making inference requests to a model called `iris`.
```

Comment:
explain 1`iris`  schema as it is used in the example inferences below?

Reply:
By `schema` do you mean the data schema or the model definition/k8s manifest?

Reply:
I meant the data schema as the inference payload uses it and perhaps it is not very clearly why the payload is constructed as such for new users coming of v2 protocol?

Reply:
I meant the data schema as the inference payload uses it and perhaps it is not very clearly why the payload is constructed as such for new users coming of v2 protocol?

Reply:
Fixed in https://github.com/SeldonIO/seldon-core/pull/4481/commits/685ca37821f0e1da416fa40fdc9cc3b10a51383d